alexandria/core/tests/test_validation.py

```diff
@@ -21,17 +21,12 @@ def validate(self, data, context):
 
     ValidatorsConfig.register_handler_class(TestValidator)
 
-    url = reverse("document-detail", args=[document.pk])
+    url = reverse("tag-detail", args=[tag.pk])
 
     # first, ensure validator is called for Document
     admin_client.patch(url, json={})
     assert call_counter["validate"] == 1
 
     # See if the validation had some effect
-    document.refresh_from_db()
-    assert document.created_by_group == "foobar"
-
-    # second, ensure validator is not called for File
-    url = reverse("file-detail", args=[file.pk])
-    admin_client.patch(url, json={})
-    assert call_counter["validate"] == 1
+    tag.refresh_from_db()
+    assert tag.created_by_group == "foobar"
```

Comment:
What was the need for this change?

Reply:
Because the test was previously for `Document` but since we already have custom validations there this tests triggers both the validations from the test and the one i added.